.mvn/jvm.config

```diff
@@ -1 +1,7 @@
 -XX:+IgnoreUnrecognizedVMOptions
```

Comment:
we're missing the Apache 2.0 license header here

Reply:
We cant add a license header to this file. I believe the rat plugin is aware of that, because it hasnt complained so far.

Reply:
I see, I didn't check that it's restricted - fair enough.